ontology/yaml/resources/HVAC/entity_types/ADY.yaml

```diff
@@ -29,3 +29,16 @@ ADY_BS_SS:
   - control_mode
   - schedule_run_command
   - control_status
+
+AION_BS_SS:
```

Comment:
Is this a stand-alone piece of equipment? If so, it will need to be defined with a new general type (AION may work) but you have defined it here in the ADY.yaml file. This would need to be moved into its own file called AION.yaml.

If this isn't stand-alone (I assume this is probably the ionizer in an AHU) then it should be defined as part of that equipment. You might suggest something like: 
- `ionizing_filter_run_command`
- `ionizing_filter_run_status`

and then create a separate `IFSS` abstract type for ionizing filter start/stop.

Reply:
on checking, it is stand-alone. I'll update again with your suggestions. thanks sir

Reply:
out of curiosity, where does this device live? is it in a duct (separate from the AHU)? does it stand in a space on its own? how does it fit into the system?